service/src/schemas/input.graphql

```diff
@@ -34,17 +34,17 @@ input AddQuizInput {
     name: String!
     description: String!
     duration: Int!
```

Comment:
can we make all of these non-compulsory (remove all of the `!`)

Reply:
only in this file btw

Reply:
Okay I copied exactly what you did from the other branch so should be good